docs/OnTrack/UI Enhancement/ComponentReviewCreateUnitModal.md

```diff
@@ -0,0 +1,40 @@
+# Component Review
+
+## Student Name: Gaganjeet Singh
+
+## Student ID: 220032936
+
+## Term: T1 2023 – SIT 378
+
+## Ontrack Component review check
+
+First select a component to review from the list below:
+
+[https://deakin365.sharepoint.com/:x:/r/sites/ThothTech2/Shared%20Documents/OnTrack%20-%20UI%20Enhancement/T3_2022/Management%20%5BT3_2022%5D/selected_task(revised).xlsx?d=wac02013da5224c958ac60fd96fac7b20&csf=1&web=1&e=UfCmZ5](<https://deakin365.sharepoint.com/:x:/r/sites/ThothTech2/Shared%20Documents/OnTrack%20-%20UI%20Enhancement/T3_2022/Management%20%5BT3_2022%5D/selected_task(revised).xlsx?d=wac02013da5224c958ac60fd96fac7b20&csf=1&web=1&e=UfCmZ5>)
+
+## Component Name
+
+\*\*\*\*./src/app/admin/modals/create-unit-modal/create-unit-modal.coffee
+
+**File Name:**create-unit-modal.coffee
+
+## Component purpose
+
+## It is used to create a new unit
+
+## Component outcomes/interactions
+
+It creates a new unit. ![image](Images/CreateUnitModal.png)
+
+**Component migration Check list** – What is needs to be checked for this component to work once
```

Comment:
Rewrite the instruction here... eg: When working, this component needs to demonstrate the following:

then create a checklist:

[ ] ability to collect details from the user
[ ] succeeds when data is valid
[ ] handles errors - duplicate unit code in the teaching period, or invalid dates
[ ] created unit is shown on success